class.jetpack.php

```diff
@@ -560,6 +564,19 @@ private function __construct() {
 		}
 	}
 
+	function rest_api_init() {
+		register_rest_route( 'jetpack/v1', '/verify_registration', array(
```

Comment:
Most Jetpack endpoints are currently versioned as `jetpack/v4` for the prefix -- not v1